src/book/03-guides/01-main-codebase/01-getting-started.mdx

```diff
@@ -175,24 +220,31 @@ To build the code, open a terminal and change into the NUbots code directory. Th
 2. With the target platform selected, choose the binaries (called "roles") that you want to build. You do this by running the following command:
 
    ```bash
+   ./b configure
    ./b configure -i
    ```
 
-   This will show a menu to choose roles to build. Enable the roles you want to build and disable the ones you don't by highlighting them using using the <kbd>↑</kbd> and <kbd>↓</kbd> arrow keys and pressing the spacebar to toggle on and off.
+   Note that you will need to do `configure` without `-i` the very first time you do this. Using `-i` will show a menu that will allow you to choose the roles to build. Enable the roles you want to build and disable the ones you don't by highlighting them using using the <kbd>↑</kbd> and <kbd>↓</kbd> arrow keys and pressing the spacebar to toggle on and off.
+
+   If you are following the keyboardwalkfake example, you will need to make sure `keyboardwalkfake` is set to `true`. Find the list of items that are prefixed with `ROLE_` and set all to `false` except for `keyboardwalkfake`, which should be `true`. Do not change any other lines unless you know what you are doing.
```

Comment:
```suggestion
   If you are following the keyboardwalkfake example, you will need to make sure `keyboardwalkfake` is set to `true`. Find the list of items that are prefixed with `ROLE_` and set all to `false` except for `keyboardwalkfake`, which should be `true`. Do not change other lines unless you want to build a different role.
```

Reply:
This was more about the build options than the other roles

Reply:
ahh true. I was just thinking that this might spook people a bit much, without telling them what they need to be spooked about. This means that they might know everything they need to know, but they wouldn't know that they did, so they might still be spooked (I am having a hard time articulating this in a way that makes sense...).

Kind of like how we frame `./b shell`, as something you [should basically never do](https://nubook.nubots.net/system/foundations/build-system#shell), even though it's quite handy a lot of the time. I have no idea why it's such a big deal, and since the documentation about it doesn't say why we should be careful, I still don't know if I am missing something. Although, I have used it a bunch, and it's really handy.

Reply:
Haha yeah sure. Maybe just something like 'You do not need to change any other build options'.